### DISCUSSION TITLE
Has anyone found a VGA->HDMI converter that worked?

### DISCUSSION THREAD

**Discussion by joelburton** [slu4coder/Minimal-64x4-Home-Computer#19] [General]
A cheap Amazon converter shows the image, but it "jumps" every second or two, which makes it unpleasant to read.

Borrowing a friend's OSSC converter, but I can't get that to show anything (but I'm not well-versed in technical details of VGA, so it may be that I'm missing an obvious setting).

I know the board works --- when connected to a LCD display with an actual VGA-in, it works fine. I just don't own such a display, nor do I have much room for it on my bench.

**Comment by slu4coder** [comment]
Hi Joel,
early VGA monitors used to accept pretty large tolerances within each of the (one or few) supported video modes. As I've demonstrated with my two older monitors, this work well with the Minimal 64x4 VGA signal. I suspect that modern VGA-to-HDMI converters are too "smart" in that they try to discriminate the Minimal 64x4 VGA signal against a larger variety of (say 15?) different video modes with each mode having tight tolerances. These devices as well as modern TVs might get confused detecting the mode and thus show the behavior you see. I'd try and look for a device that lets you manually set video modes and lets you disable "auto-detection". I'd highly appreciate someone sharing his or her experience with converting the Minimal 64x4 to HDMI, too.

**Comment by joelburton** [comment]
After some experimentation, I found that this works, right out of the box: https://www.amazon.com/dp/B0B4B2CM9F?psc=1&ref=ppx_yo2ov_dt_b_product_details

(the downside is: it's not at all inexpensive, but at least I can use my lab HDMI monitor!)

**Comment by victorliu** [comment]
Another data point: SG-VC9900 based converter boards do not work, such as this one:
https://www.amazon.com/Pasuihcay-SG-VC9900-Conversion-Board-Solution/dp/B0D5YF25Y6/ref=sr_1_1?crid=2FY2C0IOPYKQD&dib=eyJ2IjoiMSJ9.NGmOyCT8ZsUOI8w53toO0o4kNGaqujq4DOKLCdQ-bWfGjHj071QN20LucGBJIEps.mZWP35v3NRfMpnlcWbYxUx5zLLxUcXcyyBAbPtOVq1E&dib_tag=se&keywords=vc9900&qid=1743798699&sprefix=vc9900%2Caps%2C185&sr=8-1
